src/js/models/v2/aboutTheData/BaseAgencyRow.js

```diff
@@ -10,7 +10,7 @@ const BaseAgencyRow = {
     populate(data) {
         this._name = data.agency_name || '';
         this._abbreviation = data.abbreviation || '';
-        this._code = data.code || '';
+        this._code = data.agency_code || '';
```

Comment:
If this model is used for multiple endpoints, we should make sure `code` vs. `agency_code` is consistent between endpoints, or check for either.

Reply:
Good call. The `BaseAgencyRow` is only used by the Agencies page and only when on the the first tab.  That route returns `agency_code`.

I reviewed the API contracts for `/reporting/agencies` and both those routes return `agency_code`. The routes under `reporting/agencies/{agency_code}` do not return a `code` or `agency_code`